code/game/objects/items/weapons/surgery_tools.dm

```diff
@@ -128,24 +151,60 @@
 
 
 /obj/item/weapon/scalpel/laser
-	heat_production = 0
+	icon_state = "stabslash"
 	damtype = "fire"
 	sharpness_flags = SHARP_TIP | SHARP_BLADE | HOT_EDGE
 	var/cauterymode = 0 //1 = cautery enabled
+	var/obj/item/weapon/scalpel/laser/held
+
+/obj/item/weapon/scalpel/laser/New()
+	..()
+	icon_state = "scalpel"
```

Comment:
Why is this set in `New()` and not in the type definition?

Reply:
Laser scalpels have an on and off icon when used and that's what it's used for.

Reply:
That doesn't answer my question. Setting the starting `icon_state` in the type definition does not prevent you from changing the `icon_state` later.
I don't see the reason to set an `icon_state` in the `New()`.